lib/smf/smf.c

```diff
@@ -18,14 +18,12 @@ struct internal_ctx {
 	bool new_state : 1;
 	bool terminate : 1;
 	bool exit      : 1;
+	bool handled   : 1;
 };
 
-static bool share_paren(const struct smf_state *test_state,
-			const struct smf_state *target_state)
+static bool share_paren(const struct smf_state *test_state, const struct smf_state *target_state)
 {
-	for (const struct smf_state *state = test_state;
-	     state != NULL;
-	     state = state->parent) {
+	for (const struct smf_state *state = test_state; state != NULL; state = state->parent) {
```

Comment:
please run clang-format

Reply:
I did, and that was the result, plus messing up all the struct definitions.

Is there some magic I need to invoke on Windows other than `clang_format -i smf.c` as recommended by the contribution guidelines?

Reply:
On recommendation of the Discord, I've reverted all clang-format changes.

Reply:
> please run clang-format

Is the update satisfactory?

Reply:
Looks good to me